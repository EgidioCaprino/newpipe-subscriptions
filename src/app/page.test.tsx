import { render } from "@testing-library/react";
import HomePage from "./page";

it("renders the home page unchanged", () => {
  const { container } = render(<HomePage />);
  expect(container).toMatchSnapshot();
});
