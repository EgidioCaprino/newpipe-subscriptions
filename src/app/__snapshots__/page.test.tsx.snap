// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home page unchanged 1`] = `
<div>
  <header
    class="flex justify-end gap-8 p-8 text-slate-500"
  >
    <a
      href="https://github.com/EgidioCaprino/newpipe-subscriptions/issues/new"
    >
      Support
    </a>
    <a
      href="https://github.com/EgidioCaprino/newpipe-subscriptions"
    >
      Github
    </a>
  </header>
  <main
    class="flex flex-col items-center content-center space-y-8 p-24 font-sans"
  >
    <h1
      class="text-xl"
    >
      From Invidious to NewPipe
    </h1>
    <p
      class="text-center"
    >
      Load the subscriptions file you exported from Invidious
    </p>
    <input
      class="file:border file:border-solid"
      type="file"
    />
  </main>
</div>
`;
